frame/staking/src/pallet/mod.rs

```diff
@@ -1616,6 +1616,28 @@ pub mod pallet {
 			Self::chill_stash(&stash);
 			Ok(())
 		}
+
+		/// Force all validators to have at least the minimum commission. This will not affect
+		/// validators who already have a commission greater than or equal to the minimum.
+		///
+		/// Note that parameter `_validator_count` should be the upper bound of the number of
+		/// validators stored and is only used for weights.
+		#[pallet::weight(T::WeightInfo::force_apply_min_commission(*_validator_count))]
+		pub fn force_apply_min_commission(
+			origin: OriginFor<T>,
+			_validator_count: u32,
```

Comment:
There is no guarantee that the caller passes the correct value here, right?  
But since it is only root callable, this is probably not a problem.